metrics/wiki_split/wiki_split.py

```diff
@@ -0,0 +1,352 @@
+from collections import Counter
+
+import sacrebleu
+import sacremoses
+
+import datasets
+
+
+_CITATION = """
+@inproceedings{xu-etal-2016-optimizing,
+    title = {Optimizing Statistical Machine Translation for Text Simplification},
+    authors={Xu, Wei and Napoles, Courtney and Pavlick, Ellie and Chen, Quanze and Callison-Burch, Chris},
+    journal = {Transactions of the Association for Computational Linguistics},
+    volume = {4},
+    year={2016},
+    url = {https://www.aclweb.org/anthology/Q16-1029},
+    pages = {401--415
+},
+@inproceedings{post-2018-call,
+    title = "A Call for Clarity in Reporting {BLEU} Scores",
+    author = "Post, Matt",
+    booktitle = "Proceedings of the Third Conference on Machine Translation: Research Papers",
+    month = oct,
+    year = "2018",
+    address = "Belgium, Brussels",
+    publisher = "Association for Computational Linguistics",
+    url = "https://www.aclweb.org/anthology/W18-6319",
+    pages = "186--191",
+}
+"""
+
+_DESCRIPTION = """\
+WIKI_SPLIT is the combination of three metrics SARI, EXACT and SACREBLEU
+It can be used for evaluate performance performance of generated sentences
+"""
+
+
+_KWARGS_DESCRIPTION = """
+Calculates sari score (between 0 and 100) given a list of source and predicted
+sentences, and a list of lists of reference sentences.
+Args:
+    sources: list of source sentences where each sentence should be a string.
+    predictions: list of predicted sentences where each sentence should be a string.
+    references: list of lists of reference sentences where each sentence should be a string.
+Returns:
+    sari: sari score
+    sacrebleu: sacrebleu score
+    exact: exact score
+Examples:
+    >>> sources=["About 95 species are currently accepted ."]
+    >>> predictions=["About 95 you now get in ."]
+    >>> references=[["About 95 species are currently known ."]]
+    >>> wiki_split = datasets.load_metric("wiki_split")
+    >>> results = wiki_split.compute(sources=sources, predictions=predictions, references=references)
+    >>> print(results)
+    {'sari': 21.805555555555557, 'sacrebleu': 14.535768424205482, 'exact': 0.0}
+"""
+
+
+def normalize_answer(s):
+    """Lower text and remove punctuation, articles and extra whitespace."""
+
+    def remove_articles(text):
+        regex = re.compile(r"\b(a|an|the)\b", re.UNICODE)
+        return re.sub(regex, " ", text)
+
+    def white_space_fix(text):
+        return " ".join(text.split())
+
+    def remove_punc(text):
+        exclude = set(string.punctuation)
+        return "".join(ch for ch in text if ch not in exclude)
+
+    def lower(text):
+        return text.lower()
+
+    return white_space_fix(remove_articles(remove_punc(lower(s))))
+
+
+def compute_exact(a_gold, a_pred):
+    return int(normalize_answer(a_gold) == normalize_answer(a_pred))
+
+
+def compute_em(predictions, references):
+    scores = [any([compute_exact(ref, pred) for ref in refs]) * 1 for pred, refs in zip(predictions, references)]
```

Comment:
```suggestion
    scores = [any([compute_exact(ref, pred) for ref in refs]) for pred, refs in zip(predictions, references)]
```

Nit don't think you need to convert boolean to int here - it's done automatically when using `sum(...)`